recipe/meta.yaml

```diff
@@ -81,10 +86,17 @@ requirements:
 # It seems that the apply_parallel tests can have relatively small errors on windows
 {% set tests_to_skip = tests_to_skip + " or test_apply_parallel_rgb" %}  # [win]
 
+# skip known failure on Windows PyPy builds
+{% set tests_to_skip = tests_to_skip + " or test_hough_ellipse_zero_angle or test_hough_ellipse_non_zero" %}  # [win and (python_impl == 'pypy')]
+
 test:
   requires:
     - pytest
     - pytest-localserver
+    # 2021/12/04: 0.19.0: hmaarrfk
+    # We noticed that osx had strange failures with cythonized code
+    # Testing with full datasets on OSX to ensure we don't have bugs
```

Comment:
I think we need to remove this comment, or at least update it.

Reply:
you mean, just remove "on OSX" from the last line?

Reply:
Perhaps; also on the first line.  Maybe just `# Include pooch to ensure full test coverage` or something.

Reply:
Okay, let's just let the current round of tests finish and then we can make that change prior to merging